var/spack/repos/builtin/packages/superlu-dist/package.py

```diff
@@ -86,6 +86,7 @@ def install(self, spec, prefix):
             'FORTRAN      = {0}'.format(self.spec['mpi'].mpif77),
             'F90FLAGS     = -O2',
             'LOADER       = {0}'.format(self.spec['mpi'].mpif77),
+            'INCLUDEDIR  = $(SuperLUroot)/include',
```

Comment:
Can you add a space to line up the equals signs? Also, I don't see why we're creating an _empty_ list and then extending it instead of simply creating a list.